README.md

```diff
@@ -20,12 +20,11 @@ TLA+ is a formal specification and verification language to help engineers desig
 
 ## WebSites
 
-* Homepage: http://www.tlapl.us  (https://lamport.azurewebsites.net/tla/tla.html)
```

Comment:
www also works now. Not sure why, when I've checked these they weren't working for me. This specific domain I've also checked from my vm hosted in aws datacenter. Probably a transient issue with the hosting (tla.msr-inria.inria.fr)...

Reply:
FYI: tla.msr-inria.inria.fr does not host the content at lamport.azurewebsites.net

Reply:
That server hosts the html redirect from the tlapl.us domain to Lamport's website

Reply:
You're right; I had forgotten about the redirect setup.  It would be good to migrate these redirects to the LF:

```apache
kuppe@tla:/etc/apache2/sites-enabled$ cat www.tlapl.us.conf
<VirtualHost *:80>
	ServerName www.tlapl.us
        ServerAlias tlapl.us

	Redirect / http://lamport.azurewebsites.net/tla/tla.html

	ServerAdmin webmaster@localhost
	DocumentRoot /var/www/html/

	ErrorLog ${APACHE_LOG_DIR}/error.log
	CustomLog ${APACHE_LOG_DIR}/access.log combined

</VirtualHost>

<IfModule mod_ssl.c>
    <VirtualHost *:443>
    ServerAdmin webmaster@localhost
    ServerName www.tlapl.us
    ServerAlias tlapl.us

    Redirect /	http://lamport.azurewebsites.net/tla/tla.html
    DocumentRoot /var/www/html/

    ErrorLog ${APACHE_LOG_DIR}/error.log
    CustomLog ${APACHE_LOG_DIR}/access.log combined

    SSLEngine on
    SSLCertificateFile    /var/lib/dehydrated/certs/www.tlapl.us/cert.pem
    SSLCertificateKeyFile /var/lib/dehydrated/certs/www.tlapl.us/privkey.pem
    SSLCertificateChainFile /var/lib/dehydrated/certs/www.tlapl.us/fullchain.pem
    </VirtualHost>
</IfModule>

# vim: syntax=apache ts=4 sw=4 sts=4 sr noet
```

Reply:
I now remember that there were a few outages of tla.msr-inria.inria.fr recently.